src/main/java/core/basesyntax/DifferentColors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum DifferentColors {

```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
public enum Color {
```

Reply:
not fixed